python/setup.py

```diff
@@ -64,6 +64,7 @@ def get_instantiation_cu() -> List[str]:
     (root / prefix).mkdir(parents=True, exist_ok=True)
 
     group_sizes = os.environ.get("FLASHINFER_GROUP_SIZES", "1,4,6,8").split(",")
+    page_sizes = os.environ.get("FLASHINFER_PAGE_SIZES", "1,8,16,32").split(",")
```

Comment:
`1,16,32` or `1,8,16,32`? `DISPATCH_PAGE_SIZE` in `utils.cuh` only have `1,16,32`.

Reply:
I suppose keeping `1,16,32` would be fine (because `8` is a seldomly used page size). 
btw, you don't have to align with `utils.cuh` (we will refactor this file later), the macros defined there are only used in benchmarks/tests, not in pytorch packages.